index.html

```diff
@@ -0,0 +1,138 @@
+                    </div>
+                    <div class="team-layout__info">
+                        <div class="team-layout__name">Julian Jameson</div>
+                        <div class="team-layout__job">Profession</div>
+                        <div class="team-layout__socials">
+                            <i class="bi bi-facebook"></i>
+                            <i class="bi bi-instagram"></i>
+                            <i class="bi bi-twitter"></i>
+                        </div>
+                    </div>
+                </div>
+                <div class="team-layout__card">
+                    <div class="team-layout__img-container">
+                        <img src="./img/team2.svg" alt="" class="team-layout__img">
+                    </div>
+                    <div class="team-layout__info">
+                        <div class="team-layout__name">Julian Jameson</div>
+                        <div class="team-layout__job">Profession</div>
+                        <div class="team-layout__socials">
+                            <i class="bi bi-facebook"></i>
+                            <i class="bi bi-instagram"></i>
+                            <i class="bi bi-twitter"></i>
+                        </div>
+                    </div>
+                </div>
+                <div class="team-layout__card">
+                    <div class="team-layout__img-container">
+                        <img src="./img/team3.svg" alt="" class="team-layout__img">
+                    </div>
+                    <div class="team-layout__info">
+                        <div class="team-layout__name">Julian Jameson</div>
+                        <div class="team-layout__job">Profession</div>
+                        <div class="team-layout__socials">
+                            <i class="bi bi-facebook"></i>
+                            <i class="bi bi-instagram"></i>
+                            <i class="bi bi-twitter"></i>
+                        </div>
+                    </div>
+                </div>
+                <div class="team-layout__card">
+                    <div class="team-layout__img-container">
+                        <img src="./img/team4.svg" alt="" class="team-layout__img">
+                    </div>
+                    <div class="team-layout__info">
+                        <div class="team-layout__name">Julian Jameson</div>
+                        <div class="team-layout__job">Profession</div>
+                        <div class="team-layout__socials">
+                            <i class="bi bi-facebook"></i>
+                            <i class="bi bi-instagram"></i>
+                            <i class="bi bi-twitter"></i>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </main>
+
+    <footer class="footer">
+        <div class="footer__title">Newsletter</div>
```

Comment:
Don't create new blocks reuse similar blocks and modifiers. Like I showed ypou above